development/compiling/optimizing_for_size.rst

```diff
@@ -63,16 +209,30 @@ a lot of them:
 
 ::
 
-    scons p=windows target=release tools=no module_arkit_enabled=no module_assimp_enabled=no module_bmp_enabled=no module_bullet_enabled=no module_camera_enabled=no module_csg_enabled=no module_dds_enabled=no module_enet_enabled=no module_etc_enabled=no module_gdnative_enabled=no module_gridmap_enabled=no module_hdr_enabled=no module_jsonrpc_enabled=no module_mbedtls_enabled=no module_mobile_vr_enabled=no module_opensimplex_enabled=no module_opus_enabled=no module_pvr_enabled=no module_recast_enabled=no module_regex_enabled=no module_squish_enabled=no module_svg_enabled=no module_tga_enabled=no module_theora_enabled=no module_tinyexr_enabled=no module_upnp_enabled=no module_vhacd_enabled=no module_vorbis_enabled=no module_webm_enabled=no module_webp_enabled=no module_webrtc_enabled=no module_websocket_enabled=no module_xatlas_unwrap_enabled=no
+    scons p=<platform> target=release tools=no \
+    module_bmp_enabled=no module_bullet_enabled=no module_camera_enabled=no module_csg_enabled=no \
+    module_dds_enabled=no module_enet_enabled=no module_etc_enabled=no module_gdnative_enabled=no \
+    module_gridmap_enabled=no module_hdr_enabled=no module_jsonrpc_enabled=no module_mbedtls_enabled=no \
+    module_mobile_vr_enabled=no module_opensimplex_enabled=no module_opus_enabled=no module_pvr_enabled=no \
+    module_recast_enabled=no module_regex_enabled=no module_squish_enabled=no module_svg_enabled=no \
+    module_tga_enabled=no module_theora_enabled=no module_tinyexr_enabled=no module_upnp_enabled=no \
+    module_vhacd_enabled=no module_vorbis_enabled=no module_webm_enabled=no module_webrtc_enabled=no \
+    module_websocket_enabled=no module_webxr_enabled=no module_xatlas_unwrap_enabled=no
```

Comment:
I don't really like that part of the docs. Many users just copy it without trying to understand what it does, and end up disabling things that they actually need (and then report bugs about it not working).

Overall I think this page should be refactored / rewritten to give more high level explanations of what can be done to reduce binary size, and then redirect users to https://godot-build-options-generator.github.io/ to actually get explanations on what they're enabling or disabling.